manim/mobject/geometry.py

```diff
@@ -807,22 +850,98 @@ def __init__(self, *args, **kwargs):
 
     @property
     def base(self):
+        r"""The base point of the arrow tip.
+        
+        This is the point connecting to the arrow line.
+
+        Examples
+        --------
+
+        ::
+            >>> arrow = Arrow(np.array([0, 0, 0]), np.array([2, 0, 0]), buff=0)
+            >>> arrow.tip.base.round(2)
+            array([1.65, 0.  , 0.  ])
+        
+        """
         return self.point_from_proportion(0.5)
 
     @property
     def tip_point(self):
+        r"""The tip point of the arrow tip.
+        
+        Examples
+        --------
+
+        ::
+            >>> arrow = Arrow(np.array([0, 0, 0]), np.array([2, 0, 0]), buff=0)
+            >>> arrow.tip.tip_point.round(2)
+            array([2., 0., 0.])
+        
+        """
         return self.points[0]
 
     @property
     def vector(self):
+        r"""The arrow tip vector spanning from base to tip.
```

Comment:
Suggestion: "The vector with tail at the base of the arrow, and head at the tip of the arrow." (The word "span" has a precise meaning in linear algebra that may be confusing to some of our mathematically-adept readers (such as myself))

Reply:
While I don't really see the danger of interpreting this as the span in a vector space sense, especially given that it is used as a verb with specified start and end point, I don't mind rewording. I wanted to avoid tail and head though, so I propose a construction using "pointing" in 5528cc30a351ff57a5b06f3ab085620148cc2fe8.